python/make_wheels.py

```diff
@@ -0,0 +1,144 @@
+"""
```

Comment:
Do you mind running the formatter on this?

Reply:
Also, do you mind adding some explanation of ~ all of this ~ into the Python readme?

Reply:
Done!